.ci/docker-compose.yml

```diff
@@ -18,20 +18,67 @@ services:
     networks:
       - esnet
     depends_on:
-      - elasticsearch
-  elasticsearch:
+      - elasticsearch-oss
+
+  client-commercial:
+    image: docker.elastic.co/clients/elasticsearch-js:${NODE_JS_VERSION:-8}
+    build:
+      context: ..
+      dockerfile: .ci/Dockerfile
+      args:
+          NODE_JS_VERSION: ${NODE_JS_VERSION:-8}
+    environment:
+      - "TEST_ES_SERVER=http://elastic:secret@elasticsearch-commercial:9201"
+      - "ELASTIC_PASSWORD=secret"
+    volumes:
+      - ..:/usr/src/app
+      # This will mount the node_modules directory
+      # to the host machine using the buildtime directory.
+      - /usr/src/app/node_modules
+      - esvol:/tmp
```

Comment:
don't you need these two lines in the `client-oss`?

Reply:
Sorry, I fear I'm missing your point. Can you rephrase?